mir-ci/mir_ci/wayland/screencopy_tracker.py

```diff
@@ -85,11 +87,15 @@ def _frame_buffer(self, frame, format: int, width: int, height: int, stride: int
             shm_pool = self.shm.create_pool(fd, buffer_size)
             libc.close(fd)
             self.buffer = shm_pool.create_buffer(0, width, height, stride, format)
+            self.flags = ScreencopyTracker.FRAME_FLAGS(0)
```

Comment:
If I'm reading [the protocol](https://wayland.app/protocols/wlr-screencopy-unstable-v1#zwlr_screencopy_frame_v1:event:flags) right, we've been doing it wrong - only sending the `invert_y` flag once?

But with https://github.com/canonical/mir/issues/4086, we're no longer sending it at all, so maybe we should be resetting the flags on every frame?

Reply:
Nah. Needed to double-buffer the flags instead.